### PR TITLE
I AM DONE RYAN

### PR DESCRIPTION
**Pull request by alex45101**
I am done Ryan...Please no code review

### CODE REVIEW

File: Secret/Ryan.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#This is the meaning of life
+whatIsTheMeaningOfLife = 42
+
+#This is for the students :D
+#The current time of this comment is 18:32 1/2/2019
+#I wrote the time stamp the proper cause Swah > Swag
+print("Welcome Great Minds Student")
+input("Does Ryan ride a motorcycle????\n=>")
```
Comment (RScopio) [inline Secret/Ryan.py]: the correct term is wheelie machine

File: Secret/Ryan.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#Telling the user the answer so they know for next time :D
+for i in range(1000):
+    #String.Format("{0}) No", i + 1); -For Micheal because he asked what this is
+    #C# code        ^^^^^^^^^   
+    #Python code    ⌄⌄⌄⌄⌄⌄⌄⌄⌄
+    print("{i}) No".format(i=i+1))
+
+#Alex is def better than Ryan
+#lol Alex is SWAH
+gucci = input("Is Alex better than Ryan?????\n=>")
```
Comment (RScopio) [inline Secret/Ryan.py]: Impossible question to ask

File: Secret/Ryan.py (hunk 1 above)
Comment (alex45101) [reply Secret/Ryan.py]: No one cares Ryan
```suggestion
input("Does Ryan ride a thing with two wheels????\n=>")
```

File: Secret/Ryan.py (hunk 2 above)
Comment (alex45101) [reply Secret/Ryan.py]: Facts do not care about your feelings...

**Pull request merged**